### PR TITLE
Adding woosh sound for every data update

### PR DESCRIPTION
**Pull request by avivbueno**

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -46,6 +46,27 @@ function buildTimes() {
...
+      soundFile = "low-ding.mp3";
+      break;
+    case 3:
+      soundFile = "woosh.mp3";
+      break;
+  }
+  if (soundFile == "not found") return;
+
+  var audio = new Audio("./audio/" + soundFile);
+  audio.play();
```
Comment (eladg) [inline src/js/main.js]: This is a memory leak. 

Every time we will play a sound we will leak more and more memory, after the page is open for 15min you have . You do not need to create an audio element to play it, you need to play the element (which already exist) when you wish.

**Review by eladg — CHANGES_REQUESTED** [review]
Fix memory like

**Comment by avivbueno** [comment]
Ok, Is this a good fix for the issue?

**Review by eladg — APPROVED** [review]
OK, merging this and adding a new task for audio library

**Pull request merged**